src/stringBuffer.js

```diff
@@ -24,7 +24,15 @@
  * @return {function}
  */
 function makeBuffer() {
-  // write code here
+  let buffer = '';
+  return function(value) {
+    if (value !== undefined) {
+      buffer += value;
+      return buffer;
+    } else {
```

Comment:
do not need else if have return inside if

Reply:
Also, you can use the default parameter as in version with clear

Reply:
done